packages/realm-server/main.ts

```diff
@@ -283,50 +283,18 @@ if (distURL) {
   process.exit(1);
 });
 
-function getRealmPermissions(realmUrl: string) {
-  let userPermissions = {} as {
-    [realmUrl: string]: { users: RealmPermissionsInterface };
-  };
-  let userPermissionsjsonContent;
-
-  if (['development', 'test'].includes(process.env.NODE_ENV || '')) {
-    userPermissionsjsonContent = fs.readFileSync(
-      `.realms.json.${process.env.NODE_ENV}`,
-      'utf-8',
-    );
-  } else {
-    userPermissionsjsonContent = process.env.REALM_USER_PERMISSIONS;
-    if (!userPermissionsjsonContent) {
-      throw new Error(
-        `REALM_USER_PERMISSIONS env var is blank. It should have a JSON string value that looks like this:
-          {
-            "https://realm-url-1/": {
-              "users":{
-                "*":["read"],
-                "@hassan:boxel.ai":["read", "write"],
-                ...
-              }
-            },
-            "https://realm-url-2/": { ... }
-          }
-        `,
-      );
+// In case there are no permissions in the database, seed the realm with default permissions:
+// Base realm: read permissions for everyone
+// Other realms: read permissions for everyone, read/write permissions for signed up users
+async function seedRealmPermissions(dbAdapter: PgAdapter, realmURL: URL) {
+  if (!(await permissionsExist(dbAdapter, realmURL))) {
+    if (realmURL.href === 'https://cardstack.com/base/') {
+      await insertPermissions(dbAdapter, realmURL, { '*': ['read'] });
+    } else {
+      await insertPermissions(dbAdapter, realmURL, {
+        '*': ['read'],
+        users: ['read', 'write'],
```

Comment:
i thought by default realms were not readable by the world, no?

Reply:
The above reflects what we currently have in staging. All 3 realms there are publicly readable (however under basic auth), and writable by logged in users. Do you suggest we remove the public read in our deployed realms?

Reply:
hmmm, so i guess that I'm trying to square our usages of the words "draft" and "published" with our permissions for these realms. I would expect that the draft realm is only readable by users in that realm, and that the "published" realm is readable by the world.

what do others think?

Reply:
I suggest we discuss in our boxel meeting tomorrow and create tasks for new defaults.

Reply:
@jurgenwerk the decision from the team meeting is that the default should be "read/write for initial owner and indexing bot only"